plugins/clipboard/plugin.js

```diff
@@ -2539,6 +2580,227 @@
 			return undefined;
 		}
 	};
+
+	/**
+	 * Fallback dataTransfer object which is used together with {@link CKEDITOR.plugins.clipboard.dataTransfer}
+	 * for browsers supporting Clipboard API, but not supporting custom MIME types (Edge 16+ - #962).
+	 *
+	 * @since 4.8
+	 * @class CKEDITOR.plugins.clipboard.fallbackDataTransfer
+	 * @constructor Creates a class instance.
```

Comment:
TBH I feel like "Creates a class instance." brings no value, because this is simply what constructor is meant to do. So in this case i think it might be simply empty.